patches/README.md

```diff
@@ -7,15 +7,6 @@ The files in this folder are patches for packages we depend on within the repo.
 
 Each patch is described here, along with any relevant links to issues or PRs and any additional relevant details.
 
-### engine.io-client
-
-This patch updates the `exports` field in package.json to be correct for TypeScript projects using
-`moduleResolution: node16`.
-
-It applies the changes covered in this PR: https://github.com/socketio/engine.io-client/pull/711
-
-As soon as a version of the package is released with the changes in that PR, this patch can be removed.
-
```

Comment:
Should we keep some version of this, since the 4.7.2 patch seems to still be necessary?

Reply:
Woops, yep, I misread.